Source/Ansible/ansible-inventory.env

```diff
@@ -92,7 +92,15 @@ _t4dSrcAnsibleSortGroups(){
 ###
 ###### DOCUMENTATION BLOC #####
 ########### CODE BLOC #########
-    cat "$1" | cut -d '|' -f1 | rev | grep -Eo '>.*' | rev | sort | uniq | sed 's|>$||g'
+    local _t4dErrorCode=1
+    local _File=$1
+
+    if [[ -e $_file ]];then
+        cat "$_File" | cut -d '|' -f1 | rev | grep -Eo '>.*' | rev | sort | uniq | sed 's|>$||g'
+    else
+        _t4dDebugLog $perror "empty file"
```

Comment:
Update log -> File >$_File< Does not exist. 

You need to be sure that user (or even you in 3 months) will be able to understand what's going wrong